### PR TITLE
net-mail/mu: bump to 1.4.1

### PR DESCRIPTION
**Pull request by 00-matt**
Signed-off-by: Matt Smith <matt@offtopica.uk>
Package-Manager: Portage-2.3.99, Repoman-2.3.22

### CODE REVIEW

File: net-mail/mu/mu-1.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools elisp-common
+
+DESCRIPTION="Set of tools to deal with Maildirs, in particular, searching and indexing"
+HOMEPAGE="http://www.djcbsoftware.nl/code/mu/ https://github.com/djcb/mu"
```
Comment (charIes17) [inline net-mail/mu/mu-1.4.ebuild:9]: https://djcbsoftware.nl/code/mu/

Comment (00-matt) [reply net-mail/mu/mu-1.4.ebuild]: Thanks. Changed the homepage URL and manual URL in info message in all mu ebuilds.

File: net-mail/mu/mu-1.4.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	local myeconfargs=(
+		$(use_enable emacs mu4e)
+		$(use_enable mug gtk)
+		$(use_enable mug webkit)
```
Comment (juippis) [inline net-mail/mu/mu-1.4.1.ebuild:41]: Should the 'mug' USE be renamed as 'gtk'? I think it'd make more sense, then it's controllable profile-wide easier.

Comment (00-matt) [reply net-mail/mu/mu-1.4.1.ebuild:41]: I'm not sure, the documentation suggests that mug is just a toy and shouldn't be packaged. Not the most useful of tools

**Pull request closed without merging**